### PR TITLE
Fixes for Serverless and RDS turorial

### PR DESCRIPTION
**Pull request by cdaveaws**
Issue #, if available:

Description of changes:
1. Changing DB subnet name all lowercase.
2. Changing security group name to all lowercase
3. Fixing syntax for rds-pods.yaml ( added container and image name and fixed)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/content/docs/tutorials/rds-example.md [hunk 1]
```diff
@@ -230,30 +230,39 @@ apiVersion: v1
...
+     command:
+        - sleep
+        - "3600"
+     imagePullPolicy: IfNotPresent
+     env:
+      - name: PGHOST
+        valueFrom:
+         configMapKeyRef:
+          name: ${AURORA_INSTANCE_CONN_CM}
+          key: "${APP_NAMESPACE}.${AURORA_DB_INSTANCE_NAME}-host"
```
Comment (jkatz) [inline docs/content/docs/tutorials/rds-example.md:247]:
```suggestion
          name: ${RDS_INSTANCE_CONN_CM}
          key: "APP_NAMESPACE.${RDS_INSTANCE_NAME}-host"
```

File: docs/content/docs/tutorials/rds-example.md [hunk 2]
```diff
@@ -230,30 +230,39 @@ apiVersion: v1
...
+      - name: PGPORT
+        valueFrom:
+         configMapKeyRef:
+          name: ${AURORA_INSTANCE_CONN_CM}
+          key: "${APP_NAMESPACE}.${AURORA_DB_INSTANCE_NAME}-port"
```
Comment (jkatz) [inline docs/content/docs/tutorials/rds-example.md:252]:
```suggestion
          name: ${RDS_INSTANCE_CONN_CM}
          key: "APP_NAMESPACE.${RDS_INSTANCE_NAME}-port"
```

File: docs/content/docs/tutorials/rds-example.md [hunk 3]
```diff
@@ -230,30 +230,39 @@ apiVersion: v1
...
+      - name: PGUSER
+        valueFrom:
+         configMapKeyRef:
+          name: ${AURORA_INSTANCE_CONN_CM}
+          key: "${APP_NAMESPACE}.${AURORA_DB_INSTANCE_NAME}-user"
```
Comment (jkatz) [inline docs/content/docs/tutorials/rds-example.md:257]:
```suggestion
          name: ${RDS_INSTANCE_CONN_CM}
          key: "APP_NAMESPACE.${RDS_INSTANCE_NAME}-user"
```

File: docs/content/docs/tutorials/rds-example.md [hunk 4]
```diff
@@ -230,30 +230,39 @@ apiVersion: v1
...
+      - name: PGPASSWORD
+        valueFrom:
+          secretKeyRef:
+           name: "ack-creds"
```
Comment (jkatz) [inline docs/content/docs/tutorials/rds-example.md:261]:
```suggestion
           name: "${RDS_INSTANCE_NAME}-password"
```

**Review by jkatz — CHANGES_REQUESTED** [review]

File: docs/content/docs/tutorials/rds-example.md [hunk 5]
```diff
@@ -230,30 +230,39 @@ apiVersion: v1
...
+     env:
+      - name: PGHOST
+        valueFrom:
+         configMapKeyRef:
           name: ${RDS_INSTANCE_CONN_CM}
-          key: "default.${RDS_INSTANCE_NAME}-host"
-    - name: PGPORT
-      valueFrom:
-        configMapKeyRef:
+          key: "APP_NAMESPACE.${RDS_INSTANCE_NAME}-host"
```
Comment (jkatz) [inline docs/content/docs/tutorials/rds-example.md:247]:
```suggestion
          key: "${APP_NAMESPACE}.${RDS_INSTANCE_NAME}-host"
```

File: docs/content/docs/tutorials/rds-example.md [hunk 6]
```diff
@@ -230,30 +230,39 @@ apiVersion: v1
...
+      - name: PGPORT
+        valueFrom:
+         configMapKeyRef:
           name: ${RDS_INSTANCE_CONN_CM}
-          key: "default.${RDS_INSTANCE_NAME}-port"
-    - name: PGUSER
-      valueFrom:
-        configMapKeyRef:
+          key: "APP_NAMESPACE.${RDS_INSTANCE_NAME}-port"
```
Comment (jkatz) [inline docs/content/docs/tutorials/rds-example.md:252]:
```suggestion
          key: "${APP_NAMESPACE}.${RDS_INSTANCE_NAME}-port"
```

File: docs/content/docs/tutorials/rds-example.md [hunk 7]
```diff
@@ -230,30 +230,39 @@ apiVersion: v1
...
+        valueFrom:
+         configMapKeyRef:
           name: ${RDS_INSTANCE_CONN_CM}
-          key: "default.${RDS_INSTANCE_NAME}-user"
-    - name: PGPASSWORD
-      valueFrom:
-        secretRef:
-          name: "${RDS_INSTANCE_NAME}-password"
-          key: password
+          key: "APP_NAMESPACE.${RDS_INSTANCE_NAME}-user"
```
Comment (jkatz) [inline docs/content/docs/tutorials/rds-example.md:257]:
```suggestion
          key: "${APP_NAMESPACE}.${RDS_INSTANCE_NAME}-user"
```

**Review by jkatz — CHANGES_REQUESTED** [review]

**Review by jkatz — APPROVED** [review]

**Review by jaypipes — APPROVED** [review]
Sensible changes @cdaveaws thank you!

**Pull request merged**